docs/supported_ops.rst

```diff
@@ -1,44 +1,54 @@
 Supported operations
 ====================
-- **CPU** operator means that the operator can be scheduled on the CPU. Outputs of CPU operators may be used as regular inputs and to provide per-sample parameters for other operators through tensor arguments.
-- **GPU** operator means that the operator can be scheduled on the GPU. Their outputs may only be used as regular inputs for other GPU operators and pipeline outputs.
+
+Here is a list of the supported operations:
+
+- | **CPU** operator means that the operator can be scheduled on the CPU.
+  | The outputs of CPU operators may be used as regular inputs and to provide per-sample parameters
+    for other operators through tensor arguments.
+- **GPU** operator means that the operator can be scheduled on the GPU. Their outputs can only be used as
+  regular inputs for other GPU operators and pipeline outputs.
 - **Mixed** operator means that the operator accepts input on the CPU while producing the output on the GPU.
-- **Sequences** means that the operator can work (produce or accept as an input) sequence (video like) kind of input.
+- **Sequences** means that the operator can produce or accept as an input a sequence, or video type of input.
 - **Volumetric** means that the operator supports 3D data processing.
 
-How to read this doc
+Reading this guide
 ^^^^^^^^^^^^^^^^^^^^
 
-DALI Operators are used in two steps - creating the parametrized Operator instance using
-its constructor and later invoking its `__call__` operator
-in :meth:`~nvidia.dali.pipeline.Pipeline.define_graph` method of the Pipeline.
+DALI operators are used to create the parametrized operator instance by using
+its constructor and later to invoke its `__call__` operator
+in the pipeline’s :meth:`~nvidia.dali.pipeline.Pipeline.define_graph` method.
```

Comment:
This misses the point. The old documentaation was quite explicit about what's important here: the two step process.
```suggestion
DALI operators are used in two steps:

1. Parameterizing the operator with ``__init__``.
2. Invoking the parameterized operator like a function (effectively invoking its ``__call__`` method) in pipeline's :meth:`~nvidia.dali.pipeline.Pipeline.define_graph` method.
```
Perhaps we should provide a reference to functional API at this point.